schemas/hotelSchema.js

```diff
@@ -0,0 +1,56 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+const JoiPhoneNumer = Joi.extend(require('joi-phone-number'))
+import { Room } from './roomSchema'
+import { clientRateSchema } from './rateSchema'
+
+const hotelSchema = new mongoose.Schema({
+  ownerId: {
```

Comment:
I think this should be required and look at other fields because I think more should be marked as required.